Author: Pino Toscano <pino@debian.org>
Description: Disable the tests during build
 Revert the upstream forced KDE4_BUILD_TESTS to off, so tests are not built.
 .
 The tests cannot be run from the build directory as they are anyway,
 since some require to be installed, and also with the other kde daemons
 running.
Forwarded: not-needed
Last-Update: 2014-05-31

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -26,7 +26,7 @@ endif(NOT APPLE)
 configure_file (config.h.cmake ${CMAKE_CURRENT_BINARY_DIR}/config.h)
 
 # enforce build of tests
-set (KDE4_BUILD_TESTS "ON" CACHE "BOOL" "Enable building of tests" FORCE)
+set (KDE4_BUILD_TESTS "OFF" CACHE "BOOL" "Enable building of tests" FORCE)
 
 # KDE/QT definitions we want
 add_definitions (${QT_DEFINITIONS} ${KDE4_DEFINITIONS})
